parastell/invessel_build.py

```diff
@@ -460,8 +446,11 @@ def _vmec2xyz(self, poloidal_offset=0):
         """
         return self.scale * np.array(
             [
-                self.vmec_obj.vmec2xyz(self.s, theta, self.phi)
-                for theta in (self.theta_list + poloidal_offset)
+                [
```

Comment:
Need to update the docstring to match this new behavior

Reply:
I'll be rolling this branch back a bit to preserve our lofting functionality, instead adding this spline surface capability as an additional feature. That will involve restoring the `Rib` class and this method will maintain its old behavior.